services/web/server/src/simcore_service_webserver/long_running_tasks.py

```diff
@@ -7,11 +8,15 @@
 )
 from servicelib.aiohttp.long_running_tasks.server import setup
 from servicelib.aiohttp.typing_extension import Handler
+from servicelib.long_running_tasks.task import Namespace
 
+from . import redis
 from ._meta import API_VTAG
 from .login.decorators import login_required
 from .models import AuthenticatedRequestContext
 
+_LONG_RUNNING_TASKS_NAMESPACE: Final[Namespace] = "webserver-legacy"
```

Comment:
legacy?? what is this meaning?

Reply:
I'm only alligning with what we have, there is the concept of "legacy" long running tasks in the webserver. So I've kept the convension.